Sources/SwiftSyntaxBuilder/Buildables.swift.gyb

```diff
@@ -281,5 +281,61 @@ extension ${protocol} {
   }
 }
 
+%   end
+%   if 'ExpressibleAsMemberDeclBlock' in conformances:
+extension ${protocol} {
+  public func createMemberDeclBlock() -> MemberDeclBlock {
+    MemberDeclBlock(members: self)
+  }
+}
+
+%   end
+%   if 'ExpressibleAsPatternBindingList' in conformances:
+extension ${protocol} {
+  public func createPatternBindingList() -> PatternBindingList {
+    PatternBindingList([self])
+  }
```

Comment:
We have a lot of those. 
I think we should try to generate them all, so we don't have to add them in the [protocolsMap.py](https://github.com/apple/swift/blob/main/utils/gyb_syntax_support/protocolsMap.py)

Reply:
If we could make these conformances more generic so we don’t have to spell all of them out, I think that would be a big win. I suppose it also shouldn’t be too hard but haven’t thought about it in detail yet.

Reply:
I haven't thought about how to make it yet. 
Some quick thoughts: 

1: We need to identify all Collections children, like `PatternBindingList` -> `ExpressibleAsPatternBinding`.
2: When looping through all nodes that is not a SyntaxCollection, we need to see if it's child to a SyntaxCollection.
2: a: If it's not a child ignore
2: b: If it's a child, then we need to add default conformance to the SyntaxCollection

Reply:
Sound good to me. I suppose in that case we don’t need to explicitly specify the collection-element to collection conformances (e.g. `ExpressibleAsConditionElement` to `ExpressibleAsConditionElementList`) in `protocolsMap.py`. 

And we could probably also generate the remaining protocol extensions because they always follow the same pattern:
```swift
extension <ConformingType> {
   public func create<ConfomringToType>() -> <ConformingToType> {
     <ConformingToType>(<someLabel>: self)
   }
 }
```

Reply:
PR in #334